vcpkg/TOC.yml

```diff
@@ -144,6 +144,8 @@
     href: concepts/registries.md
   - name: Script ports
     href: maintainers/authoring-script-ports.md
+  - name: System package managers

```

Comment:
```suggestion
  - name: Using vcpkg compared to system package managers

```

Reply:
How about:

"Comparing vcpkg with system package managers like apt"

?

Reply:
I agree with your suggestion, but I think we can remove the "like apt".